editor/editor_node.cpp

```diff
@@ -2099,6 +2099,29 @@ void EditorNode::edit_item(Object *p_object) {
 	}
 }
 
+void EditorNode::store_object_properties(Object *p_object) {
+	ERR_FAIL_NULL(p_object);
+	if (ScriptInstance *si = p_object->get_script_instance()) {
```

Comment:
Lets avoid assignments in if conditions, they look really confusing.

Reply:
:(